curp/src/server/bg_tasks.rs

```diff
@@ -507,8 +507,15 @@ async fn bg_election<
 ) {
     let last_rpc_time = state.map_read(|state_r| state_r.last_rpc_time());
     let candidate_timeout = *timeout.candidate_timeout();
+    // FIXME: block the server from election in the first time it starts. It should be removed when persistency is added
```

Comment:
The reason why you do this can be more straightforward.  That's because the voted-for message will be lost after recovery without persistent storage.

Reply:
I have re-thought about it and remove the code here. I don't find it necessary to block the server from starting an election. Stoping the server from voting is enough.